test/test.js

```diff
@@ -1,27 +1,50 @@
 function readBlobAsText(blob) {
   return new Promise(function(resolve, reject) {
-    var reader = new FileReader()
-    reader.onload = function() {
-      resolve(reader.result)
-    }
-    reader.onerror = function() {
-      reject(reader.error)
+    // FileReader is not available on Workers in Firefox, but FileReaderSync
+    // is.
+    var reader
+    if (typeof FileReaderSync !== "undefined") {
+      reader = new FileReaderSync()
+      try {
+        resolve(reader.readAsText(blob))
+      } catch(e) {
+        reject(e);
+      }
+    } else {
+      reader = new FileReader()
+      reader.onload = function() {
+        resolve(reader.result)
+      }
+      reader.onerror = function() {
+        reject(reader.error)
+      }
+      reader.readAsText(blob)
     }
-    reader.readAsText(blob)
   })
 }
 
 function readBlobAsBytes(blob) {
   return new Promise(function(resolve, reject) {
-    var reader = new FileReader()
-    reader.onload = function() {
-      var view = new Uint8Array(reader.result)
-      resolve(Array.prototype.slice.call(view))
-    }
-    reader.onerror = function() {
-      reject(reader.error)
+    var reader
+    if (typeof FileReaderSync !== "undefined") {
```

Comment:
this should use single quotes (in both places) to fix the JSHInt warnings which are making the build fail